neuralmonkey/nn/projection.py

```diff
@@ -19,9 +19,12 @@ def linear(inputs, size, scope="LinearProjection"):
         A tensor of shape batch x size
     """
     with tf.variable_scope(scope):
-        return tf.nn.seq2seq.linear(inputs, size, True)
+        # TODO this will be soon deprecated, should be rewrited with
+        # tf.contrib.layers.fully_connected and a support for list of tensors
+        return tf.nn.seq2seq.linear(inputs, size, True, scope=scope)
 
 
+# SUGGESTION: what about using ReLu as a default? It have better results
```

Comment:
To je nevhodně formovaná otázka - chci říct, že žiju v domění, že v knihovnách je zvykem mít u těchto věcí `tanh` jako default, takže dávat tam něco jinýho by mohlo být zdrojem problémů spojených s reprodukcí :-)